embrace-android-api/api/embrace-android-api.api

```diff
@@ -179,6 +179,27 @@ public abstract interface class io/embrace/android/embracesdk/internal/api/UserA
 	public abstract fun setUsername (Ljava/lang/String;)V
 }
 
+public abstract interface class io/embrace/android/embracesdk/internal/capture/activity/OpenEvents {
```

Comment:
I should add doc to more clearly articulate the design - I was going to do that before submitting this for review, but (I thought) the PR messiness with the trunk change had forced my hand.

So, the idea is that the set of events to create a trace is a super set of the lifecycle events and spans multiple invocations of activity opening. And how they are fired for the lifecycle events also depends on the Android version. This set of events effectively normalizes all those difference and combines both the lifecycle and non-lifecycle events (i.e. the render stuff). 

The listener for these events will be responsible for starting and stoping spans, so they only have to be aware of these, instead of all the messiness of what this intends to encapsulate and normalize. This design will make each component more easily testable, and tease out the logic and complexity.

Reply:
`embrace-android-features` or `embrace-android-core` will hide it from prying eyes